src/smokeTest/java/uk/gov/hmcts/futurehearings/pip/smoke/common/rest/RestClient.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.futurehearings.pip.smoke.common.rest;
+
+import io.restassured.response.Response;
+
+import java.util.Map;
+
+import static io.restassured.RestAssured.given;
+
+public class RestClient {
+
+    public static Response makeGetRequest(final Map<String, String> headers,
+                                          final String authorizationToken,
+                                          final String rootContext) {
+        return given()
```

Comment:
Suggest: align the return block code to be consistent with other test types e.g. functional and acceptance tests

Reply:
This is correct? As no switch statement?